#3 Create orders table.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE orders(
+o_id INT,

```

Comment:
In my opinion, `order_id` will be better because is more descriptive than `o_id` and still, you can always use an [alias](https://www.w3schools.com/sql/sql_alias.asp) in queries if you want.

Reply:
Or maybe `orderid` will be even better because it's good practice to stick to one convention. In users, you already use `userid` as a name for the id column.